pymobiledevice3/remote/utils.py

```diff
@@ -27,6 +27,9 @@ def stop_remoted_if_required() -> None:
         return
 
     remoted = get_remoted_process()
+    if remoted is None:
+        yield
```

Comment:
The `stop_remoted_if_required()` is not a context-amanger. You should simply return here